_widget/src/components/app/component.vue

```diff
@@ -93,8 +93,8 @@ export default {
     q () {
       this.chooseRoute();
 
-      if (typeof this.$refs.body?.selectFirstArticle === 'function') {
-        this.$refs.body.selectFirstArticle();
+      if (typeof this.$refs.body?.selectNoArticle === 'function') {
```

Comment:
Some context as to why I introduced this originally (in case it wasn't clear): after changing my search query, it felt strange that the selected item was still the one from the previous query.

Reply:
That makes sense